tests/upgrade/test_crossgrade.sh

```diff
@@ -430,12 +472,12 @@ checkEchosrv
 
 # Run internal traffic in the background since we may have to relaunch it if the job fails.
 sendInternalRequestTraffic &
-sendExternalRequestTraffic "${INGRESS_ADDR}"
+sendExternalRequestTraffic "${INGRESS_ADDR}" &   # TODO: if we wait this to finish, all the following steps will suceed.
```

Comment:
if we wait then this is not really testing much, The whole point is that the traffic is sent during the update

Reply:
yes, that's why I have the `&` here to make the external traffic send during both installation and upgrade in background. Here is the issue related to this script. https://github.com/istio/istio/issues/18369